storage/storage.go

```diff
@@ -424,14 +513,59 @@ func (storage DBStorage) WriteReportForCluster(
 		return err
 	}
 
+	deleteQuery := "DELETE FROM rule WHERE org_id = $1 AND cluster_id = $2;"
```

Comment:
Why do you delete if you have upsert in next few lines?

Reply:
Because we should remove previous results before adding new one

Reply:
What's the purpose of upsert then?

Reply:
Insert data to DB